initfiles/bin/init_roxie

```diff
@@ -40,9 +40,6 @@ fi
 export LIBC_FATAL_STDERR_=1
 export restarts=0
 
-ulimit -c unlimited
-ulimit -n $NUM_ROXIE_HANDLES
```

Comment:
Ideally these limits would both be configurable across the system (defaults).
Plus, per component (which we had for dafilesrv and roxie before this commit).

But, the dafilesrv and roxie limits look like they set these limits unconditionally (I presume via defaults in the xsd's), so could end up *lowering* the limits from system defaults, which wasn't good.
[ I'm judging that from the fact that the ulimit -n $NUM_ROXIE_HANDLES or ulimit -c $handlelimit was executed unconditionally] 
Really should only set at all if user has configured specific value.